core/modules/system/system.module

```diff
@@ -2030,6 +2030,15 @@ function system_library_info() {
     ),
   );
 
+  // Announce.
+  $libraries['announce'] = array(
```

Comment:
Because this is a Backdrop-custom library, I think we should prefix this as
```suggestion
  $libraries['backdrop.announce'] = array(
```
That would line up with the other Backdrop-provided JS libraries. We should also move this array up before all the jQuery libraries.

Reply:
Done